build/ci/do_bats.sh

```diff
@@ -195,11 +220,13 @@ do_process_branch() {
 #            else
 #                ./"$file" ${BATSOPT} || log_nonzero_rc "$file" $?
 #            fi
+        # The prefix will not be considered if the timeout was not set
         if [[ -n "$RUN_COVERAGE" ]]; then
-            tmp_out="$(kcov --include-path="${SOURCES}" "$out_coverage" "$file" -p ${BATSOPT})" || do_count_failures $?
+            tmp_out="$( ${timeout_cmd} kcov --include-path="${SOURCES}" "$out_coverage" "$file" -p ${BATSOPT})" || do_count_failures $?
         else
-            tmp_out="$(./"$file" -p ${BATSOPT})" || do_count_failures $?
+            tmp_out="$( ${timeout_cmd} ./"$file" -p ${BATSOPT})" || do_count_failures $?
         fi
+        echo "$tmp_out"
         tmp_exit_code=$?
```

Comment:
You capture the exit code of echo. Invert the line.
Then the echo should be removed after testing